sql/core/src/test/resources/sql-tests/results/group-by.sql.out

```diff
@@ -338,7 +338,7 @@ SELECT corr(DISTINCT x, y), corr(DISTINCT y, x), count(*)
 -- !query schema
 struct<corr(DISTINCT x, y):double,corr(DISTINCT y, x):double,count(1):bigint>
 -- !query output
-1.0	1.0	3
+0.9999999999999999	0.9999999999999999	3
```

Comment:
I guess `0.9999999999999999` is incorrect.

Reply:
The result is not incorrect, it is just a precision issue with double. For example,
```java
2 / Math.sqrt(2 * 2) = 1.0
2 / Math.sqrt(2) * Math.sqrt(2) = 0.9999999999999999
```
 From the user's perspective, 1.0 is more user-friendly. 
I am currently unsure about whether to sacrifice user-friendliness in order to support an extreme case.

Reply:
You can reference the output of other mainstream databases.

Reply:
@beliefer Different mainstream databases have both of these calculation formulas. But I now believe that there is no need to modify Spark's code for this extreme case because Spark's formula can easily solve for finite decimals.
Thanks, I will close this PR.